dashboards/edx.erb

```diff
@@ -28,12 +28,12 @@ Dashing.numColumns = 12
       data-moreinfo="Python code coverage %"
       data-min="0" data-max="100" style="background-color:#1aa1e0"></div>
     </li>
-    <li data-row="1" data-col="9" data-sizex="2" data-sizey="3">
+<!--     <li data-row="1" data-col="9" data-sizex="2" data-sizey="3">
       <div data-id="bok_coverage" data-view="Meter" data-title="Acceptance"
       data-moreinfo="Bok-choy code coverage %"
       data-min="0" data-max="100" style="background-color:#1aa1e0"></div>
     </li>
-    <li data-row="1" data-col="11" data-sizex="2" data-sizey="3">
+ -->    <li data-row="1" data-col="9" data-sizex="2" data-sizey="3">
```

Comment:
I prefer to have this open tag on a new line from the close comment, at the same indentation level as the other list items in this list, but that's a minor readability detail. So, up to you. Otherwise :+1: